app/views/observations/by_login.html.haml

```diff
@@ -0,0 +1,155 @@
+- content_for(:title) do
+  = @title = is_me?( @selected_user ) ? t(:edit_observations) : t( :observations_by_user, user: h(@login) )
+
+- content_for(:extracss) do
+  = stylesheet_link_tag 'observations/by_login', 'observations/filters'
+  /[if lte IE 7]
+    = stylesheet_link_tag 'blueprint/ie', 'observations-ie'
+
+- content_for(:extrajs) do
+  = google_maps_js
+  :javascript
+    $(document).ready( function( ) {
+      // render map
+      $("#map").taxonMap({ appendMarkerToList: true, placement: "observations-login" });
+      window.map = $("#map").data("taxonMap");
+      // Reveal columns
+      $('.created_at').show();
+    });
+  = javascript_include_tag 'observations/by_login_bundle', 'map_bundle', 'observations/by_login'
+
+- content_for(:extrahead) do
+  %link{ rel: "alternate", type: "application/atom+xml",
+    title: t( :recent_atom_observations_by, :login => @login ),
+    href: url_for( controller: 'observations',
+    action: 'by_login', login: @login, format: 'atom' ) }
+  
+  %meta{ property: "og:title", content: t( :observations_by_user, user: @login ) }
+  %meta{ property: "og:url", content: request.original_url }
+  %meta{ property: "og:description", content: t( :users_latest_observations_on_site, user: @login, site: @site.name ) }
+  - photo_observations = (@observations || []).select{|o| o.observation_photos_count > 0}.sort_by{|o| [o.comments_count.to_i, o.id]}[0..3]
+  - if photo_obs = photo_observations.last
+    - if photo = photo_obs.observation_photos.map(&:photo).detect{|p| p.medium_url =~ /^http/}
+      %meta{ property: "og:image", content: @shareable_image_url = photo.medium_url }
+      %meta{ name: "twitter:image", content: @shareable_image_url }
+  - if @observer_provider_authorizations && twitter_pa = @observer_provider_authorizations.detect{|pa| pa.provider_name == "twitter"}
+    %meta{ name: "twitter:creator:id", content: twitter_pa.provider_uid }
+  - if photo_observations.size > 0
+    %meta{ name: "twitter:card", content: "gallery" }
+    - photo_observations.each_with_index do |o,i|
+      - next unless photo = o.observation_photos.map(&:photo).detect{|p| p.medium_url =~ /^http/}
+      %meta{ name: "twitter:image#{i}", content: photo.medium_url }
+  - else
+    %meta{ name: "twitter:card", content: "summary" }
+
+- unless @proj_obs_errors_obs.blank?
+  .error
+  %h3
+    = raw t(:the_following_observations_could, :link => link_to(@project.title, @project))
+    %ul
+      - for o in @proj_obs_errors_obs
+        %li.clear.stacked
+          = image_and_content observation_image(o, :style => "max-width: 48px; margin: 0 auto; display:block;") do
+            = link_to o.to_plain_s(:user => false), o
+            %div
+              = raw @proj_obs_errors[:errors][o.id].join("<br/>")
+
+.bootstrap
+  .container
```

Comment:
Could you make this `.container-fluid` instead? I know this screws up alignment if you're navigating between pages in the subnav, but I think it looks pretty broken when the observations are fluid but the header is not.

Reply:
Yeah, I was going for consistency between pages - but after seeing it like this now also prefer how it looks when viewing the page by itself. (Commit added)